app.py

```diff
@@ -26,6 +26,6 @@ def database_disconnect(response):
 api.add_resource(item.ItemsResource, '/items/')
 api.add_resource(item.ItemResource, '/item/<uuid:uuid>')
 api.add_resource(UsersResource, '/users/')
-api.add_resource(UserResource, '/users/<uuid:uuid>')
+api.add_resource(UserResource, '/users/<uuid:uuid>', methods=['put', 'delete'])
```

Comment:
Questo non dovrebbe servire a niente. In flask restful i verbi degli endpoint delle risorse sono specificati dal nome del metodo.